OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnDeploy.cs

```diff
@@ -132,7 +132,7 @@ Activity IWrapMove.WrapMove(Activity moveInner)
 				return moveInner;
 
 			var activity = new DeployForGrantedCondition(self, this, true);
-			activity.Queue(self, moveInner);
+			activity.Queue(moveInner);
```

Comment:
This is in the wrong commit ("Make Tick return bool." instead of "Move ChildActivity handling into base Activity class.").